src/app/member/components/MemberEditPage.tsx

```diff
@@ -0,0 +1,369 @@
+"use client";
+import React from "react";
+import { Button } from "@/components/ui/button";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+import { addDays, format } from "date-fns";
+import { User, UserRole } from "@/constant/constant";
+import { onSubmit } from "./memberActions";
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from "@/components/ui/popover";
+import { cn } from "@/lib/utils";
+import { CalendarIcon } from "lucide-react";
+import { Calendar } from "@/components/ui/calendar";
+import FormWrapper from "../../../common/FormWrapper";
+import { Label } from "@/components/ui/label";
+
+const phoneValidator =
+  /^((\+92)?(0092)?(92)?(0)?)(3)([0-9]{2})((-?)|( ?))([0-9]{7})$/gm;
+const cnicValidator =
+  /^(3)([0-9]{1})([0-9]{3})((-?)|( ?))([0-9]{7})((-?)|( ?))([0-9]{1})$/gm;
+
+export const formSchema = z.object({
+  name: z
+    .string({ message: "Name is required" })
+    .min(3, "Name should have at least 3 characters"),
+  phone: z
+    .string({ message: "Phone is required" })
+    .regex(phoneValidator, "phone number is not valid"),
+  date_of_birth: z.coerce
+    .date()
+    .min(
+      new Date("01-01-1990"),
+      "You can not suitable for this job because your are over age"
```

Comment:
There's only a min age and that's 16 years old. No max age.

Reply:
When i use default age as 16 year old then the header and year in the calender does not change , and in future it will be modify along with selection of year and month